### ISSUE TITLE
prod env enable degub mode?

### ISSUE THREAD

**Issue by griffenliu** [apache/apisix-dashboard#690]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [+] Bug
- [ ] Requirements
- [ ] Feature or performance improvement
- [ ] Other

___
### Bug
- Which version of Apache APISIX Dashboard, OS, and Browser?
2.0
- What happened?
prod env enable degub mode?
```
func SetUpRouter() *gin.Engine {
	if conf.ENV != conf.EnvLOCAL && conf.ENV != conf.EnvBETA {
		gin.SetMode(gin.DebugMode)
	} else {
		gin.SetMode(gin.ReleaseMode)
	}
```

**Comment by idbeta** [comment]
Please forgive me, I don’t understand the question very well, can you provide more information?

**Comment by juzhiyuan** [comment]
This issue should cc @nic-chen

**Comment by nic-chen** [comment]
that's a bug. let's fix it in 2.0

**Comment by nic-chen** [comment]
fixed.
